src/DAL/migration/v2.1.1.sql

```diff
@@ -0,0 +1,4 @@
+--  ************************************** note ******************************************
+--        this script will fail to run if there are duplicate active jobs in db
+-- ***************************************************************************************
+ALTER TABLE public."Job" ADD CONSTRAINT UQ_uniqueness_on_active_tasks EXCLUDE ("resourceId" with =, version with =) WHERE (type = 'Discrete-Tiling' AND (status = 'Pending' OR status = 'In-Progress'));
```

Comment:
type = 'Discrete-Tiling' - i don't want this hard coded job type
should be constarint of (job type + version + id + (pending or in-progress))